.github/workflows/build-libtorch-images.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     runs-on: "${{ needs.get-label-type.outputs.label-type }}linux.9xlarge.ephemeral"
     strategy:
       matrix:
-        cuda_version: ["12.4", "12.1", "11.8"]
```

Comment:
Four CUDA versions is too much for infra and unecessary. Let's remove 12.1 in this PR.